configure.ac

```
@@ -43,8 +43,8 @@ case $host_os in
          # These Homebrew packages may be keg-only, meaning that they won't be found
          # in expected paths because they may conflict with system files. Ask
          # Homebrew where each one is located, then adjust paths accordingly.
-         valgrind_prefix=`$BREW --prefix valgrind 2>/dev/null`
-         if test x$valgrind_prefix != x; then
+         if $BREW list --versions valgrind >/dev/null; then
+           valgrind_prefix=`$BREW --prefix valgrind 2>/dev/null`
```

```suggestion
           valgrind_prefix=$($BREW --prefix valgrind 2>/dev/null)
```

See https://github.com/koalaman/shellcheck/wiki/SC2006

---

Yeah, the Bash FAQ says " `...` is the legacy syntax required by only the very oldest of non-POSIX-compatible bourne-shells." But then if we want to perform that cleanup, we should do this change in the entire file...

---

Yep. I wasn't going to make other random cleanups.